core/packages/PackageDB.cc

```diff
@@ -121,16 +135,23 @@ const std::vector<std::string> &PackageDB::extraPackageFilesDirectoryPrefixes()
     return extraPackageFilesDirectoryPrefixes_;
 }
 
+bool PackageDB::isTestFile(const core::GlobalState &gs, const core::File &file) {
```

Comment:
Hopefully in the future this will change to just check if the file is within a test package or a normal package.